core/lib/dal/migrations/20240725194916_l1_batches_consensus_committees.up.sql

```diff
@@ -0,0 +1,12 @@
+ALTER TABLE l1_batches_consensus
+ADD validator_committee JSONB NULL,
```

Comment:
How about we make it one big json, so that it is extensible to adding more information to the contract? Btw, is the contract upgradeable?

Reply:
The committees have distinct functionality, so putting them on separate columns makes sense. However they are mostly part of "data derived from the registry contract at a given batch", so it’s possible to consolidate.

Reply:
The contract is not upgradable.

Reply:
> I see it as the opposite of https://github.com/matter-labs/zksync-era/pull/2518#discussion_r1695305287. If it's not merely about copying data, but rather about extracting/setting the committees, then the columns shouldn't be unified

Agreed, I'm advocating that it should be about merely copying the data then.

Reply:
> that it should be about merely copying the data then.

but why? If we just wanted to copy - there would probably less reason to do this feature at all. We want to interpret the data and assign the committee for batch - so that the information, or method how exactly we do this assignment is not needed in other places. Just get it from the DB!

Reply:
Latest update is to have 1 column, for the attester committee.